--- picard-0.10/picard.desktop	2008-07-27 15:59:00.000000000 +0000
+++ picard-0.10/picard.desktop	2008-11-11 00:49:04.000000000 +0000
@@ -2,9 +2,10 @@
 Encoding=UTF-8
 Name=MusicBrainz Picard
 Comment=The next generation MusicBrainz tagger
+Comment[pl]=Znacznik MusicBrainz nowej generacji
 Exec=picard
 Terminal=false
 Type=Application
 StartupNotify=true
-Icon=picard-32.png
+Icon=picard
 Categories=Application;AudioVideo;AudioVideoEditing;
