diff -burNp picard-0.11/picard.desktop picard-0.11-new/picard.desktop
--- picard-0.11/picard.desktop	2008-12-01 18:27:05.000000000 +0100
+++ picard-0.11-new/picard.desktop	2009-01-15 15:41:02.254473346 +0100
@@ -1,9 +1,10 @@
 [Desktop Entry]
 Name=MusicBrainz Picard
 Comment=Tag your music with the next generation MusicBrainz tagger
+Comment[pl]=Znacznik MusicBrainz nowej generacji
 Exec=picard
 Terminal=false
 Type=Application
 StartupNotify=true
-Icon=picard-32
+Icon=picard
 Categories=AudioVideo;AudioVideoEditing;
